### DISCUSSION TITLE
CVE-2024-43639

### DISCUSSION THREAD

**Discussion by KamistixX** [latchset/kdcproxy#67] [Announcements]
Is your KKDCP implementation affected by https://msrc.microsoft.com/update-guide/vulnerability/CVE-2024-43639 ?

Or is this vulnerability specific to Windows Server ?

Thank you

**Comment by simo5** [comment]
RCEs are by they nature a product specific vulnerability that depends on how the code is written, they are not a protocol level vulnerability.

**Reply by KamistixX** [reply]
Great answer, thank you.
